c/tskit/tables.h

```diff
@@ -718,6 +718,11 @@ typedef struct {
 #define TSK_UNION_NO_CHECK_SHARED (1 << 0)
 #define TSK_UNION_NO_ADD_POP (1 << 1)
 
+/* Flags for table collection equals */
+#define TSK_IGNORE_TOP_LEVEL_METADATA (1 << 0)
+#define TSK_IGNORE_METADATA (1 << 0)
```

Comment:
Just catching up here - does the ``TSK_IGNORE_METADATA`` actually do anything now @mufernando? If not we should file an issue and remove it.

Reply:
Gah! How'd I miss that. Nice catch!

Reply:
https://github.com/tskit-dev/tskit/pull/911

Reply:
oops! I think I missed that in the rebase before merging. Thanks for catching it @jeromekelleher and for fixing it @benjeffery !